tests/test/paywall.test.js

```diff
@@ -0,0 +1,58 @@
+const url = require('../helpers/url')
+
+jest.setTimeout(30000)
+
+describe('The Unlock Paywall', () => {
+  const testLockAddress = '0x5Cd3FC283c42B4d5083dbA4a6bE5ac58fC0f0267'
+
+  function lockSelector(name) {
+    return `#${name}_${testLockAddress}`
+  }
+
+  beforeAll(async () => {
+    await page.goto(url('/dashboard'))
+    await expect(page).toMatch('Creator Dashboard')
```

Comment:
Could you explain why going to the `/dashboard` is required? Since this is served from a different app, which may eventually evolve/disappear, I wonder if this is not including coupling where we do not need it?

Reply:
literally not 1 line of that is needed, I used it to debug what was going on, and so have removed it. Because I spent days trying to get this work, I couldn't even see those lines any more (attentional blindness?)